src/metabase/models/action.clj

```diff
@@ -207,25 +207,29 @@
 
    Pass in known-models to save a second Card lookup."
   [known-models & options]
-  (let [actions                         (apply select-actions-without-implicit-params options)
-        implicit-action-model-ids       (set (map :model_id (filter (comp #(= :implicit %) :type) actions)))
-        models-with-implicit-actions    (if known-models
-                                          (->> known-models
-                                               (filter #(contains? implicit-action-model-ids (:id %)))
-                                               distinct)
-                                          (when (seq implicit-action-model-ids)
-                                            (db/select 'Card :id [:in implicit-action-model-ids])))
-        implicit-parameters-by-model-id (when (seq models-with-implicit-actions)
-                                          (implicit-action-parameters models-with-implicit-actions))]
+  (let [actions                       (apply select-actions-without-implicit-params options)
+        implicit-action-model-ids     (set (map :model_id (filter (comp #(= :implicit %) :type) actions)))
+        models-with-implicit-actions  (if known-models
+                                        (->> known-models
+                                             (filter #(contains? implicit-action-model-ids (:id %)))
+                                             distinct)
+                                        (when (seq implicit-action-model-ids)
+                                          (db/select 'Card :id [:in implicit-action-model-ids])))
+        model-id->db-id               (into {} (for [card models-with-implicit-actions]
+                                                 [(:id card) (:database_id card)]))
```

Comment:
This is the key change in this block, the rest is whitespace + renaming for consistency